bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -118,6 +121,12 @@ pub struct Client {
     pub(crate) sliding_sync_reset_broadcast_tx: Arc<SharedObservable<()>>,
 }
 
+impl fmt::Debug for Client {
+    fn fmt(&self, formatter: &mut fmt::Formatter<'_>) -> fmt::Result {
+        formatter.debug_struct("Client").field("client", &self.client).finish_non_exhaustive()
+    }
+}
```

Comment:
What is this impl for?

Reply:
It was required at some point, it's no longer needed. I'm removing them.